cli/dts/swc.d.ts

```diff
@@ -0,0 +1,1568 @@
+// Copyright 2018-2020 the Deno authors. All rights reserved. MIT license.
+
+/// <reference no-default-lib="true" />
+
+// --- AST Nodes ---
+
+export interface Span {
```

Comment:
Are these being put into the global scope or something?

Reply:
Nope, these are the AST return types but I'm unable to refer them in deno.unstable.d.ts